docs-markdown/src/controllers/preview-controller.ts

```diff
@@ -17,25 +17,29 @@ export function previewTopic() {
     reporter.sendTelemetryEvent("command", { command: telemetryCommand });
 
     const editor = vscode.window.activeTextEditor;
-
-    if (!common.isValidEditor(editor, false, "preview topic")) {
+    if (!editor) {
+        common.noActiveEditorMessage();
         return;
-    }
+    } else {
+        if (!common.isValidEditor(editor, false, "preview topic")) {
+            return;
+        }
 
-    if (!common.isMarkdownFileCheck(editor, false)) {
-        return;
-    }
+        if (!common.isMarkdownFileCheck(editor, false)) {
+            return;
+        }
 
-    const osPlatform = common.getOSPlatform();
-    const extensionName = "docsmsft.docs-preview";
-    const { msTimeValue } = common.generateTimestamp();
-    const friendlyName = "docsmsft.docs-preview".split(".").reverse()[0];
-    const inactiveMessage = `[${msTimeValue}] - The ${friendlyName} extension is not installed.`;
-    if (common.checkExtension(extensionName, inactiveMessage)) {
-        if (osPlatform === "win32") {
-            vscode.commands.executeCommand("docs.showPreviewToSide");
-        } else {
-            vscode.commands.executeCommand("markdown.showPreviewToSide");
+        const osPlatform = common.getOSPlatform();
+        const extensionName = "docsmsft.docs-preview";
+        const { msTimeValue } = common.generateTimestamp();
+        const friendlyName = "docsmsft.docs-preview".split(".").reverse()[0];
+        const inactiveMessage = `[${msTimeValue}] - The ${friendlyName} extension is not installed.`;
+        if (common.checkExtension(extensionName, inactiveMessage)) {
+            if (osPlatform === "win32") {
+                vscode.commands.executeCommand("docs.showPreviewToSide");
+            } else {
+                vscode.commands.executeCommand("markdown.showPreviewToSide");
```

Comment:
docs-preview supports cross-platform, do we still need to use vscode default markdown preview in Linux/macOS?

Reply:
@wibjorn I mean `docs-preview` has already supported cross-platform, you could give it a try :)